### ISSUE TITLE
Node.js 16 actions are deprecated

### ISSUE THREAD

**Issue by dmarcucci** [amondnet/vercel-action#249]
https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

**Comment by simon-robertson-shift** [comment]
v25 doesn't appear to support Node 18+

The following deployment error is being reported:
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: amondnet/vercel-action@v25. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

Is this likely to be resolved soon or should we look for an alternative GitHub action to use?

**Comment by dmarcucci** [comment]
I just looked into it. The solution is literally a one-line change, lol.

I'll fork and make a PR shortly.

**Comment by dmarcucci** [comment]
PR is [here](https://github.com/amondnet/vercel-action/pull/254).

@amondnet If you could please review, merge, and publish so that people can continue to use this action, that would be greatly appreciated.

**Comment by vickkhera** [comment]
> PR is [here](https://github.com/amondnet/vercel-action/pull/254).
> 
> @amondnet If you could please review, merge, and publish so that people can continue to use this action, that would be greatly appreciated.

And please update the README to specify the new version because people still copy/paste it into their action.

**Comment by TheJizel** [comment]
Fix is in v25.2.

Thanks @amondnet!

**Comment by dmarcucci** [comment]
Fix is in v25.2.

Thanks @amondnet!
